### ISSUE TITLE
How does -Exclude work on Registry queries

### ISSUE THREAD

**Issue by D-Jeffrey** [MicrosoftDocs/PowerShell-Docs#4264]
Does Exclude work for Registry paths?  Does it required wild cards?  Is there a way to have a -match type filter on the Exclude? 
Example Get-ChildItem HKCU:\Software\Microsoft\Windows\Currentversion\  -Exclude FileExt  ... or "*FileExt*"

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: afee1cf3-2e19-4a97-3794-815d54ef0097
* Version Independent ID: ffa98c5b-6e0a-dc99-94b2-2fddc3f78da4
* Content: [Get-ChildItem (Microsoft.PowerShell.Management)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-childitem?view=powershell-6)
* Content Source: [reference/6/Microsoft.PowerShell.Management/Get-ChildItem.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Management/Get-ChildItem.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @daxian-dbw
* Microsoft Alias: **dongbow**

**Comment by sdwheeler** [comment]
Yes, exclude works for registry paths. Wildcards are supported but not required. But it is important to understand that the exclusion applies to the subkeys and not to the item properties (registry values).
